requirements.txt

```diff
@@ -1,5 +1,4 @@
 Flask==1.1.1
-Werkzeug==0.16.0
```

Comment:
Also removed the explicit dependency, as it's probably better to let Flask set the version.

Reply:
I personally feel better when all dependencies are pinned to specific versions. I remember a specific version of itsdangerous (a Flask dependency) breaking compatibility overnight, as it was not pinned it broke the app on the next deployment.

Reply:
We had a similar issue with Kombu (Celery's dependency), but I'm not sure if those rare (?) issues are worth the extra work / risk of having incompatible dependency because of forgetting to upgrade one but not the other.

An approach of a lock file is probably a better one to handle such issues. This way you know what gets upgraded but need to define only top level dependencies (until an issue happens). But not sure if there is a stable/well adopted solution for Python that does this :|